Drive Scan/DriveScan.xaml.cs

```diff
@@ -324,6 +341,61 @@ public void ThemeSwitch(dynamic Sender, RoutedEventArgs e = null)
             }
         }
 
+        /// <Summary>
+        /// Load a scan
+        /// </summary>
+        public void LoadScan(object Sender, RoutedEventArgs e)
```

Comment:
This function still has the comments from when it was copied and pasted from the scanning procedure.
Maybe consider adapting them to the new purpose.